gtk4/src/expression.rs

```diff
@@ -23,9 +23,13 @@ pub unsafe trait IsExpression:
 }
 
 impl Expression {
+    pub fn is<E: IsExpression>(&self) -> bool {
+        self.type_() == E::static_type()
```

Comment:
I am wondering whether it makes sense to keep the `type_()` method as public

Reply:
I did that in a separate commit so we can only cherry-pick the first one for the next release

Reply:
Seems useful to get the type?

Reply:
I can't see where you would need that in general

Reply:
It's also not hurting to keep it :) It's potentially useful information and you can't get that any other way right now.

People are also using the `type_()` function on objects.

Reply:
I dropped that commit, I will add similar functions to Gsk.RenderNode/Gdk.Event then